### PR TITLE
Use new Surelog/ include location.

### PR DESCRIPTION
**Pull request by hzeller**
Also: use instead of a comment, use [[fallthrough]] to let the
compiler know our intent (and squash warnings due to that).

### CODE REVIEW

**Comment by hzeller** [comment]
(note, tests fail due to #305 not yet merged)

**Comment by hzeller** [comment]
Can't assign reviewer, so here manually CC'ing you @kamilrakoczy

**Review by tgorochowik — APPROVED** [review]
LGTM once the CI is green, I'll also look into adding you here so you can assign people

**Comment by hzeller** [comment]
I think the green-ness will happen after #305 is merged

**Review by kamilrakoczy — APPROVED** [review]

**Comment by hzeller** [comment]
Alright, re-based and all tests pass.
I don't have permission to merge, so one of you needs to do that.

**Comment by rkapuscik** [comment]
Thank you!

**Pull request merged**
